projects/product-configuration/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "@spartacus/product-configuration",
```

Comment:
Can we discuss the name? :) 
I don't mind it, but if we can use a shorter abbreviation, it would be nice.